apps/web/src/pages/Home/ExecutiveHomePage/ExecutiveHomePage.tsx

```diff
@@ -0,0 +1,515 @@
+              data-h2-padding="base(0)"
+              data-h2-list-style="base(none)"
+            >
+              {pools.map((pool) => (
+                <li key={pool.id}>
+                  <PoolCard pool={pool} />
+                </li>
+              ))}
+            </ul>
+          </div>
+        ) : (
+          <div
+            data-h2-padding="base(x1)"
+            data-h2-radius="base(s)"
+            data-h2-shadow="base(medium)"
+            data-h2-background="base(foreground)"
+          >
+            <Heading level="h3" size="h6" data-h2-margin-top="base(0)">
+              {intl.formatMessage({
+                defaultMessage: "More opportunities are coming soon!",
+                id: "Cia2li",
+                description:
+                  "Heading for message when there are no executive opportunities available",
+              })}
+            </Heading>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "New opportunities are posted throughout the year. By starting your profile now, you’ll be better positioned to submit a strong application when the time comes.",
+                id: "QAlCtg",
+                description:
+                  "Message displayed when there are no executive opportunities available",
+              })}
+            </p>
+          </div>
+        )}
+      </SkewedContainer>
+      <div
+        data-h2-background="base(white) base:dark(black.light)"
+        data-h2-padding="base(x3 0)"
+        data-h2-border-top="base(solid 1px gray.lighter)"
+        data-h2-margin-top="base(-x1) l-tablet(-x3)"
+        data-h2-position="base(relative)"
+        data-h2-z-index="base(3)"
+      >
+        <div data-h2-container="base(center, large, x1) p-tablet(center, large, x2)">
+          <Heading
+            level="h2"
+            Icon={PuzzlePieceIcon}
+            color="quaternary"
+            data-h2-margin-top="base(0)"
+          >
+            {intl.formatMessage({
+              defaultMessage: "What we can do for you",
+              id: "5bdLGy",
+              description: "Heading for the executive opportunities",
+            })}
+          </Heading>
+          <div
+            data-h2-display="base(grid)"
+            data-h2-grid-template-columns="base(1fr) p-tablet(repeat(2, minmax(0, 1fr))) l-tablet(repeat(3, minmax(0, 1fr)))"
+            data-h2-gap="base(x2) p-tablet(x3)"
+            data-h2-padding="base(x2, 0, 0, 0) p-tablet(x3, 0, 0, 0)"
+          >
+            <CardFlat
+              color="tertiary"
+              title={intl.formatMessage({
+                defaultMessage: "Find pre-qualified executive talent",
+                id: "1R3SYH",
+                description:
+                  "Heading for the digital government executive talent search",
+              })}
+              links={[
+                {
+                  href: "#", // Note: Waiting on email address
```

Comment:
Probably shouldn't merge without this.

Reply:
Just came in! 😄 Added in [add missing mailto link](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7294/commits/0d1a8225b43cd71c86e1a51078340c386bf6157a)